### DISCUSSION TITLE
Trouble with "All posts" section

### DISCUSSION THREAD

**Discussion by jilmun** [athul/archie#55] [Q&A] [answered]
I tried to apply this theme to a new hugo site, and I can't get the "All posts" link to work. I just have a blank site now with two dummy posts including tags. All of the other sections work as expected when served in localhost. but the "All posts" section goes to a 404 page and I can't figure out why. I'm still learning this stuff so any help is appreciated. Is there an issue with the list.html file?

**Comment by athul** [comment, marked as answer]
Hey, 
So how all posts work is on how you name the directory with all your blog posts. If you write all the posts inside a directory called `posts` inside the content directory, then if you go to *<your-site>/posts* you'll find all the posts, similarily if you write all the posts inside say a folder called `blog` inside the content directory then if you go to <your-site>/blog and you'll find all your posts.

**Reply by jilmun** [reply]
Oh my gosh, thank you so much! I made my posts into a folder called "post" instead of "posts" :(.

**Reply by athul** [reply]
Glad you could figure it out. :)
